terms/chip.md

```diff
@@ -0,0 +1,9 @@
+# Chip
```

Comment:
So, chips are a set of custom gates that can execute specific operations like hashing or pairing?